postgres/count extension/calculate--0.0.1.c

```diff
@@ -0,0 +1,71 @@
+//C program extension to count number of insert, delete, update, select queries Executed
+
+//including required header files
+#pragma once
```

Comment:
why pragma here

Reply:
inorder to prevent multiple inclusions sir

Reply:
multiple inclusions of what ?

Reply:
resolved it sir